man/man5/zfs-module-parameters.5

```diff
@@ -1765,6 +1765,20 @@ table.
 Default value: \fB131,072\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBfletcher_4_impl_set\fR (string)
+.ad
+.RS 12n
+Select a fletcher 4 implementation to use.
+Two selectors are supported: \fBfastest\fR and \fBscalar\fR. If there exists
+multiple implementation of fletcher 4, the \fBfastest\fR will choose the
+fastest algorithm to use. \fBscalar\fR is to use CPU to compute fletcher 4.
```

Comment:
This needs to be updated.  Three selectors are supported **fastest**, **scalar**, and **avx2**.